src/Stratis.Bitcoin/Connection/IConnectionManager.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace Stratis.Bitcoin.Connection
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Net;
+    using System.Threading.Tasks;
+
+    using NBitcoin;
+
+    using Stratis.Bitcoin.Configuration;
+    using Stratis.Bitcoin.Configuration.Settings;
+    using Stratis.Bitcoin.P2P;
+    using Stratis.Bitcoin.P2P.Peer;
+    using Stratis.Bitcoin.P2P.Protocol.Payloads;
+
+    public interface IConnectionManager : IDisposable
```

Comment:
If you moved this out then please add comments as well.

Reply:
I don't understand the logic behind your comment ?
If I separate the interface I have to comment it, but if it is in the same file as the class I don't ? Why ?